react-app/src/Finish/index.js

```diff
@@ -36,7 +39,12 @@ export default function Finish({ db, setReset }) {
   return (
     <PageTemplate topBarColor={ThemeColor.background} bottomColor={ThemeColor.background}>
       <Header />
-      <Info moves={STRETCHING_EXERCISE_DATA.length} time={`10`} format={`Sitting`}></Info>
+      <Info
+        moves={exercise.length}
+        // These coming two lines are kinda hard code
```

Comment:
Yes. Let's keep it like this but we can creat a ticket to track it

Reply:
We have the solution on another page but not enough time to find it out.